propagation/SWALS/examples/nthmp/BP09/test_min_stage.R

```diff
@@ -0,0 +1,48 @@
+source('../../../plot.R')
+
+
+test_min_stage = function(atol=1e-6) {
+    domain_number = 2
+    most_recent_run = rev(Sys.glob(paste0('OUTPUTS/RUN_*')))[1]
+
+
+    min_stage_raster = merge_domains_nc_grids(
+        multidomain_dir = most_recent_run, 
+        domain_index=domain_number,
+        desired_var = 'min_stage',
+        return_raster=TRUE
+    )
+    min_stage = as.matrix(min_stage_raster)
+
+    # check min minimum stage is -5.987428
+    check_min_stage = min(min_stage, na.rm=TRUE)
+    if (abs(min(check_min_stage) - -5.987428) > atol) {
```

Comment:
This test is very fragile -- any minor change to the solver will cause it to fail. Can you please make it less fragile (e.g. within a few percent is OK).

Reply:
Solved in [3c42c55](https://github.com/GeoscienceAustralia/ptha/pull/49/commits/3c42c55ca7e02277ca38b3c511d4a5304fc634e7) by using a relative tolerance of 1%